racket-custom.el

```diff
@@ -243,6 +243,13 @@ Defaults to a regexp ignoring all inputs of 0, 1, or 2 letters."
   :safe #'booleanp
   :group 'racket-repl)
 
+(defcustom racket-images-max-width -1
+  "The max width of inlined images. Negative values disable auto-scaling."
```

Comment:
Maybe add a second line, something like "Note: This option only works when Emacs is built with ImageMagick (libMagickWand) support." I'm just using the phrase from the Emacs Lisp Info manual.